src/docs/product/cli/dif.mdx

```diff
@@ -9,14 +9,12 @@ information about crash reports for most compiled platforms. `sentry-cli` can be
 used to validate and upload debug information files. For more general
 information, refer to [_Debug Information Files_](/workflow/debug-files/).
 
-<Note>
-
-Source maps, while also being debug information files, are handled differently
 
+<Note>
+Source maps, while also being debug information files, are handled
+differently in Sentry. For more information see [Source Maps in sentry-cli](/product/cli/releases/#sentry-cli-sourcemaps).
```

Comment:
Hmm, the <Note> does not seem to enjoy having links inside it.

https://sentry-docs-f3z0ufimj.sentry.dev/product/cli/dif/

Reply:
Fixed by adding spaces before and after note tag